Dockerfile

```diff
@@ -35,28 +35,14 @@ WORKDIR /go/src/github.com/docker/distribution
 # the registry is used to test both schema 1 and schema 2 manifests. Generally,
```

Comment:
Can we remove “schema 1” from this line too?

Reply:
Yup! Guess I screwed up a rebase here (or overlooked 😂)